### DISCUSSION TITLE
while EC2 configure with github action, I got this error. Please help me to solve the issue.

### DISCUSSION THREAD

**Discussion by webbariflolabs** [community/community#61516] [Actions]
Http response code: NotFound from 'POST https://api.github.com/actions/runner-registration' (Request Id: E154:515A:EEED30:FE8201:64BBC6FA)
{"message":"Not Found","documentation_url":"https://docs.github.com/rest"}
Response status code does not indicate success: 404 (Not Found).

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @webbariflolabs , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your question entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
